packages/astro/src/content/loaders/glob.ts

```diff
@@ -131,7 +131,7 @@ export function glob(globOptions: GlobOptions): Loader {
 				if (entryType.getRenderFunction) {
 					let render = renderFunctionByContentType.get(entryType);
 					if (!render) {
-						render = await entryType.getRenderFunction(settings);
+						render = await entryType.getRenderFunction(settings.config);
```

Comment:
This is the only runtime change I did, the way it was setup, it was exposing `AstroSettings` publicly, which isn't in line with what we typically do